### PR TITLE
Docs: User's guide review (section: Introduction)

### PR DESCRIPTION
**Pull request by beraldoleal**

### CODE REVIEW

**Comment by beraldoleal** [comment]
Thanks @willianrampazzo, I just force pushed to fix one error with build system.

File: docs/source/guides/user/chapters/introduction.rst [hunk 1]
```diff
@@ -206,13 +220,11 @@ them executable (`chmod +x /tmp/pass /tmp/fail)`, and running them as
...
 
-    $ avocado run --external-runner=/bin/curl http://local-avocado-server:9405/jobs/ \
-                                           http://remote-avocado-server:9405/jobs/
+    $ avocado run --external-runner=/bin/curl https://google.com/
     JOB ID     : 56016a1ffffaba02492fdbd5662ac0b958f51e11
     JOB LOG    : /home/<user>/avocado/job-results/job-<date>-<shortid>/job.log
-    (1/2) http://local-avocado-server:9405/jobs/: PASS (0.02 s)
-    (2/2) http://remote-avocado-server:9405/jobs/: FAIL (3.02 s)
-    RESULTS    : PASS 1 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0
+    (1/1) http://google.com/: PASS (0.02 s)
```
Comment (richtja) [inline docs/source/guides/user/chapters/introduction.rst:226]: Nitpick: there should be https

**Review by richtja — APPROVED** [review]
LGTM

File: docs/source/guides/user/chapters/introduction.rst (hunk 1 above)
Comment (beraldoleal) [reply docs/source/guides/user/chapters/introduction.rst]: Thanks, fixed with a force-push here.

**Comment by beraldoleal** [comment]
> Ooops, failures in:
> 
> ```
> ======================================================================
> FAIL: test_output_incompatible_setup (selftests.functional.test_output.OutputPluginTest)
> ----------------------------------------------------------------------
> Traceback (most recent call last):
>   File "/home/travis/build/avocado-framework/avocado/selftests/functional/test_output.py", line 359, in test_output_incompatible_setup
>     self.assertIsNotNone(error_regex.match(result.stderr_text),
> AssertionError: unexpectedly None : Missing error message from output:
> b'avocado run: error: argument --json: Options --xunit --json are trying to use stdout simultaneously. Please set at least one of them to a file to avoid conflicts\n'
> ----------------------------------------------------------------------
> ```

yes, was trying to fix. Solved here with a force push.

**Comment by beraldoleal** [comment]
@clebergnu ping.

**Review by clebergnu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**